querydsl-core/src/main/java/com/querydsl/core/util/JavaSyntaxUtils.java

```diff
@@ -27,63 +25,8 @@
 
     private JavaSyntaxUtils() { }
```

Comment:
Yes, maybe add the deprecation and replace the usage with SourceVersion.isKeyword